### DISCUSSION TITLE
Resources of day

### DISCUSSION THREAD

**Discussion by JDLimaco** [vkurko/calendar#158] [General]
With the "resourceTimeGridDay" property, can I give the columns a width or a minimum width?

**Comment by vkurko** [comment]
The width of the entire calendar is evenly distributed across all columns. The width of the calendar is set by the width of the container it is in.

**Reply by JDLimaco** [reply]
I understand, but in my case, for example, I need 20 resources per day and each resource looks very small
For this reason I was thinking of giving each resource a minimum width, in Full Calendar I have investigated and it can be done, but only for a group of resources, so now I am looking to see if it can be done for each one.
Giving a width to the entire container could be an option, but when I scroll the relationship between resources and hours would be lost
Could you help me with that, please?

**Reply by vkurko** [reply]
Unfortunately, there is no setting for the minimum column width. For your case, it seems like a better option to arrange resources in rows rather than columns. However the corresponding [view](https://github.com/vkurko/calendar/issues/30) has not yet been implemented.

**Reply by JDLimaco** [reply]
I understand, don't worry, thanks for the prompt response anyway :D
